src/core/events.js

```diff
@@ -1,6 +1,7 @@
 import EventEmitter from 'eventemitter2'
 import store from '../store/store'
 import * as selectors from '../store/selectors'
+import { each } from 'lodash'
```

Comment:
I would entertain the possibility of rewriting those functions locally in order to reduce the total file size, but not using them at all in favour of a more verbose and less dry approach does not seem right, in the end it's still vanilla JS. However, if "Tree Shaking" becomes a thing for Webpack, I think it's really a moot point.